### DISCUSSION TITLE
Highlight the link/bookmark mouse hover

### DISCUSSION THREAD

**Discussion by toprogramm** [Kovah/LinkAce#766] [Ideas & Feature Requests]
![image](https://github.com/Kovah/LinkAce/assets/74075594/01f78f6f-2c48-4e39-a425-74a1abb408f2)

## When I watch on my links and if I have a lot of links, it's sometimes hard to click correct link when the link doesn't have a highlight. Of course it's about mouse hover. I believe it's a good idea to add the highlight like the image below.

![image](https://github.com/Kovah/LinkAce/assets/74075594/a16a46d7-7041-4bed-9824-5c485dd016e3)

**Comment by Kovah** [comment]
I will have a look at this for version 2, which also highly optimizes the link views.

**Reply by toprogramm** [reply]
It would be great. Thanks for your work 😊😊😊
Another feature -  You can add dividers between name/link/other description  like on the picture below
![image](https://github.com/Kovah/LinkAce/assets/74075594/09102d21-2924-4fc2-b319-bfaee818a5cc)
And when user clicks on the link or the link area -  it will open this link in the new tab. 
![image](https://github.com/Kovah/LinkAce/assets/74075594/772ece49-09bc-4875-b83b-ffe9840f92fd)
